scc/carvel-package-supply-chain.hbs.md

```diff
@@ -0,0 +1,107 @@
+
+    ```yaml
+    ootb_supply_chain_basic:
+      carvel_package:
+        workflow_enabled: true
+        gitops_subpath: path/to/my/dir
+    ```
+
+3. (Optional) Set a name suffix. This will determine the suffix of the name of the Carvel Package. See the [Carvel Package template](ootb-template-reference.hbs.md#carvel-package-experimental) for more information.
+
+    ```yaml
+    ootb_supply_chain_basic:
+      carvel_package:
+        workflow_enabled: true
+        gitops_subpath: path/to/my/dir
+        name_suffix: vmware.com
+    ```
+
+Then, in `tap-values`, configure the [Out of the Box Basic Supply Chain](ootb-supply-chain-basic.hbs.md) package with your GitOps parameters, as described in [GitOps versus RegistryOps](gitops-vs-regops.hbs.md#gitops).
+
+Finally, install the [Out of the Box Basic Supply Chain](ootb-supply-chain-basic.hbs.md) package.
+
+## How To: Developer
+
+### Prerequisites
+
+- Your operator will need to enable the Carvel Package workflow for the [Out of the Box Basic Supply Chain](ootb-supply-chain-basic.hbs.md) package, as described above.
+
+### Creating a Workload
+
+To utilize the Carvel Package workflow, you must add the label `apps.tanzu.vmware.com/carvel-package-workflow=true` to your workload.
+With the `tanzu` CLI, you can do so by using the following flag:
+
+- `--label apps.tanzu.vmware.com/carvel-package-workflow=true`
+
+For example:
+
+  ```bash
+  tanzu apps workload create tanzu-java-web-app \
+    --app tanzu-java-web-app \
+    --type server \
+    --label apps.tanzu.vmware.com/carvel-package-workflow=true \
+    --image IMAGE
+  ```
+
+Expect to see the following output:
+
+  ```console
+  Create workload:
+      1 + |---
+      2 + |apiVersion: carto.run/v1alpha1
+      3 + |kind: Workload
+      4 + |metadata:
+      5 + |  labels:
+      6 + |    app.kubernetes.io/part-of: tanzu-java-web-app
+      7 + |    apps.tanzu.vmware.com/carvel-package-workflow: "true"
+      8 + |    apps.tanzu.vmware.com/workload-type: server
+      9 + |  name: tanzu-java-web-app
+     10 + |  namespace: default
+     11 + |spec:
+     12 + |  image: IMAGE
+  ```
+
+You can override two parameters set by the operator:
+
+1. (Optional) Set a GitOps subpath. This will determine the path in your GitOps repository to which Carvel Packages are written. See the [Carvel Package template](ootb-template-reference.hbs.md#carvel-package-experimental) for more information.
+
+Set this parameter by modifying `workload.spec.params.carvel_package_gitops_subpath`. With the `tanzu` CLI, you can do so by using the following flag:
+
+- `--param carvel_package_gitops_subpath=path/to/my/dir`
+
+1. (Optional) Set a name suffix. This will determine the suffix of the name of the Carvel Package. See the [Carvel Package template](ootb-template-reference.hbs.md#carvel-package-experimental) for more information.
```

Comment:
Can you describe the default suffix when they don't make use of this optional param?